File: dbflow-tests/src/androidTest/java/com/raizlabs/android/dbflow/test/structure/backup/BackupTest.java
package com.raizlabs.android.dbflow.test.structure.backup;

import com.raizlabs.android.dbflow.sql.language.Delete;
import com.raizlabs.android.dbflow.test.FlowTestCase;

/**
 * Description:
 */
public class BackupTest extends FlowTestCase {

    public void testBackup() {

        Delete.table(BackupModel.class);

        BackupModel backupModel = new BackupModel();
        backupModel.name = "Test";
        backupModel.save();

        assertTrue(backupModel.exists());

        Delete.table(BackupModel.class);

    }
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/structure/backup/BackupTest.kt
package com.raizlabs.android.dbflow.structure.backup

import com.raizlabs.android.dbflow.sql.language.Delete
import com.raizlabs.android.dbflow.FlowTestCase

import org.junit.Test

import org.junit.Assert.assertTrue

/**
 * Description:
 */
class BackupTest : FlowTestCase() {

    @Test
    fun testBackup() {

        Delete.table(BackupModel::class.java)

        val backupModel = BackupModel()
        backupModel.name = "Test"
        backupModel.save()

        assertTrue(backupModel.exists())

        Delete.table(BackupModel::class.java)

    }
}
